src/HavenoDaemon.ts

```diff
@@ -275,7 +275,7 @@ class HavenoDaemon {
    */
   async createAccount(password: string): Promise<void> {
     let that = this;
-    await new Promise(function(resolve, reject) {
+    await new Promise<void>(function(resolve, reject) {
```

Comment:
The other promises are not typed this way throughout the class. Does it compile using `resolve(void);` instead?